index.html

```diff
@@ -703,6 +760,19 @@ <h2>Register performance entry type</h2>
         <li>Append <var>type</var> to <a>supported entry types</a>.</li>
       </ol>
     </section>
+    <section data-link-for="PerformanceObserver">
+      <h2>Eligibility for adding a PerformanceEntry to a buffer</h2>
```

Comment:
Not sure we need to abstract this mini-algorithm out: it is small and simple and only called once. I would just do the steps inline in the algorithm where it is invoked. No strong opinion though...

Reply:
The reason for splitting this algorithm is clarity. Emily wanted to clarify the reasoning for running these steps, and splitting out that algorithm does that.